chainstate/src/detail/tests/double_spend_tests.rs

```diff
@@ -24,7 +24,10 @@ use common::{
     },
     primitives::{time, Amount, Id},
 };
-use crypto::random::{self, Rng};
+use proptest::prelude::*;
+
+proptest! {
+#![proptest_config(ProptestConfig::with_cases(1))]
```

Comment:
I'm just curious: does the tests execution take too long with the default value?

Reply:
~0.5s vs 0.01s previously

Reply:
I believe now it's as fast as before, right?

Reply:
yes